resources/views/filament/footer.blade.php

```diff
@@ -48,12 +48,13 @@ class="block h-7 dark:hidden"
 
     <div class="flex w-full justify-center pb-4 pt-2">
         <span class="w-11/12 text-center text-xs lg:w-3/4 xl:w-7/12">
-            © 2016-{{ date('Y') }} Canyon GBS LLC. All Rights Reserved. Canyon GBS™ and Aiding App™ are trademarks
+            © 2016-{{ date('Y') }} Canyon GBS LLC. All Rights Reserved. Canyon GBS&reg; and Aiding App&reg; are
+            trademarks
```

Comment:
Though `&reg;` is typically fine in almost all cases, lets use the numeric version `&#174;` as it has the most compatibility between the two.

Reply:
Done